modules/switcher.js

```diff
@@ -0,0 +1,28 @@
+// display current date
+/* eslint-disable no-undef */
+export const currentDate = () => {
+  const DateTime = luxon.DateTime.local();
+
+  return DateTime;
```

Comment:
Please fix the time and date displayed currently it's not human readable.

<img width="689" alt="image" src="https://user-images.githubusercontent.com/34578631/170990735-509d0bea-cf1c-48de-adf3-bfac8c6a8806.png">

**Hint:** you can use something like this to fix 👇 

```
const { DateTime } = luxon;
const now = DateTime.now();
return now.toLocaleString(DateTime.DATETIME_MED);

```